app/build.gradle

```diff
@@ -5,7 +5,7 @@ android {
     buildToolsVersion '28.0.3'
 
     defaultConfig {
-        minSdkVersion 17
+        minSdkVersion 19
```

Comment:
Does this change need this update?

Reply:
yes, the encoding stuff needs Appium settings to be at API 19, which is fine for us (it's Android 4.4)

Reply:
I've changed the code to still support API 17. The market share there is about 1.5-2%, but whatever